kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -14,12 +14,10 @@
       <mat-svg id="edit-icon" class="name-edit" category="image" name="edit" aria-hidden="true"/>
     </div>
 
-    <hr>
-
     <div class="header">
-      <h1>
+      <h2 class="table-title">
         {{$tr('tableTitle')}}
-      </h1>
+      </h2>
```

Comment:
Not just style, the text here ("Manage Learners and Coaches in this Class") is a subheading, and should not be an `h1?` (I'm not too sure how headings should work actually)

![screen shot 2017-03-20 at 3 27 19 pm](https://cloud.githubusercontent.com/assets/6668144/24124610/bea61b44-0d81-11e7-997c-44e8a00d3445.png)